### PR TITLE
feat: pinned note

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-4-10

### CODE REVIEW

File: note-app/src/Reducer/setnoteReducer.js [hunk 1]
```diff
@@ -24,14 +24,17 @@ const setnoteReducer = (state,action) => {
...
                    title: '',
                    text: '',
                  noteColor: '',
                    tags: '',
-                 priority: 'High'
+                 priority: 'High',
+                 pinned: false
                 }
          default:
           return state;
```
Comment (rohanmathur91) [inline note-app/src/Reducer/setnoteReducer.js]: throw an error saying `action type not found` instead of returning state

**Review by rohanmathur91 — COMMENTED** [review]
- Code is readable and file structuring is also good.
 - Overall looks good to merge

**Review by Aditi0505 — APPROVED** [review]
- Clean code 
- Good naming convention 
-  Please use code formatting tool (eg: prettier)  for all files.

**Review by shubhamkumar648 — APPROVED** [review]
1. good folder structure and perfect use variable name 
2. design looks nice 
3. good to merge

File: note-app/src/Reducer/setnoteReducer.js (hunk 1 above)
Comment (aniketsingh13) [reply note-app/src/Reducer/setnoteReducer.js]: will do that

**Pull request merged**